addons/website/views/website_templates.xml

```diff
@@ -2789,8 +2789,9 @@ Sitemap: <t t-esc="url_root"/>sitemap.xml
                     </ul>
                 </div>
                 <span t-if="next_step"
-                        class="d-inline-block d-md-none text-muted"
-                        t-out="'Next: ' + next_step[1]['name']"/>
+                        class="d-inline-block d-md-none text-muted">
+                        Next: <t t-out="next_step[1]['name']"></t>
```

Comment:
```suggestion
                        Next: <t t-out="next_step[1]['name']"/>
```

Reply:
Whoops, vscode is too smart for its own good. Fixed it